Adyen/Model/BrowserInfo.swift

```diff
@@ -12,6 +12,9 @@ public struct BrowserInfo: Encodable {
     
     /// The device default user-agent.
     public var userAgent: String?
+
+    /// The Accept request HTTP header indicates which content types, expressed as MIME types, the client is able to understand.
+    public var acceptHeader: String { "text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8" }
```

Comment:
spell check is failing because of `html,application` :-), you can add it to `.typokana_ignore` file as an exception.

Reply:
👍